Description: historyform: Clear rows before loading history
 This patch backports commit 07991f06fe7404708a66537ad87a2f42a8f71565.
Author: Peter Colberg <peter@colberg.org>
Forwarded: not-needed
Last-Update: 2015-12-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/gui/historyform.cpp
+++ b/src/gui/historyform.cpp
@@ -125,6 +125,8 @@ void HistoryForm::loadHistory()
 	unsigned long totalCallDuration = 0;
 	unsigned long totalConversationDuration = 0;
 
+	m_model->setRowCount(0);
+
     std::list<t_call_record> history;
 
     call_history->get_history(history);
